kubetest2-gke/deployer/up.go

```diff
@@ -192,13 +192,21 @@ func (d *Deployer) CreateCluster(project string, cluster cluster, subNetworkArgs
 	}
 
 	if d.WindowsEnabled {
-		args := d.createWindowsNodePoolCommand(project, cluster, locationArg, "windows-pool", d.WindowsImageType)
+		args := d.createNodePoolCommand(project, cluster, locationArg, "windows-pool", d.WindowsImageType, d.WindowsMachineType, d.WindowsNumNodes)
 		output, err := runWithOutputAndReturn(exec.Command("gcloud", args...))
 		if err != nil {
 			return fmt.Errorf("error creating windows node-pool: %v, output: %q", err, output)
 		}
 	}
 
+	if d.SecondNodePoolEnabled {
```

Comment:
this would then iterate over `ExtraNodePools []string` or `ExtraNodePools []NodeSpec`